GitHub-Test-DHprojects/GitHub-Test-4-PyNLPDiscuss/lyndonDiscussion.md

```diff
@@ -0,0 +1,23 @@
+
+# Lyndon's Discussion Post
+
+
+**A particular method of NLP that I found interesting was the idea of topic modeling. The idea of locating "hidden themes" with text documentaries in order to label them automatically.**
+
+As I have previous mentioned, topic modeling for documents sounds like an interesting method to try out during a class assignment or project.
+
+[Bee Movie Script](http://www.script-o-rama.com/movie_scripts/a1/bee-movie-script-transcript-seinfeld.html Links to an external site.I feel that using this script would be an interesting test to see what category this would be listed under.)
```

Comment:
@LPHerschell The link wasn't fully formed around "Bee Movie Script" in the published markdown, but it's just b/c you're missing a closing parenthesis: 

You have:
```
[Bee Movie Script](http://www.script-o-rama.com/movie_scripts/a1/bee-movie-script-transcript-seinfeld.html
```

Should be:

```
[Bee Movie Script](http://www.script-o-rama.com/movie_scripts/a1/bee-movie-script-transcript-seinfeld.html)
```